libtransmission/torrent-files.cc

```diff
@@ -322,7 +322,11 @@ namespace
 
     // Shortcut to avoid extra work below.
     // All the paths below involve filenames that begin with one of these chars
+#if defined(_WIN32) || defined(_WIN64)
     static auto constexpr ReservedFilesBeginWithOneOf = "ACLNP"sv;
+#else
+    static auto constexpr ReservedFilesBeginWithOneOf = "."sv;
```

Comment:
Why thou?
In POSIX `.` at the beginning just marks file as hidden, but name is still valid.

Reply:
It's to reserve two specific names: "." and "..". Not all of those starting with ".".